### PR TITLE
Added zero link unfurling app installation card for nodejs sample

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

**Review by Pawan-MSFT — APPROVED** [review]
looks correct, Approving...

File: samples/msgext-search-sso-config/nodejs/bots/teamsMessagingExtensionsSearchAuthConfigBot.js [hunk 1]
```diff
@@ -352,7 +356,44 @@ class TeamsMessagingExtensionsSearchAuthConfigBot extends TeamsActivityHandler {
...
+                    "size": "extraLarge"
+                    },
+                    {
+                    "type": "TextBlock",
+                    "text": "Install the app to view full content of the card.",
+                    "size": "large"
+                    }
+                ]
+            });
+
```
Comment (Pawan-MSFT) [inline samples/msgext-search-sso-config/nodejs/bots/teamsMessagingExtensionsSearchAuthConfigBot.js:380]: just remove additional spaces.

File: samples/msgext-search-sso-config/nodejs/bots/teamsMessagingExtensionsSearchAuthConfigBot.js [hunk 2]
```diff
@@ -352,7 +356,44 @@ class TeamsMessagingExtensionsSearchAuthConfigBot extends TeamsActivityHandler {
                 }
             }
         }
-        return await super.onInvokeActivity(context);
+
+        if(context.activity.name === "composeExtension/anonymousQueryLink") {
+
```
Comment (Pawan-MSFT) [inline samples/msgext-search-sso-config/nodejs/bots/teamsMessagingExtensionsSearchAuthConfigBot.js:361]: remove additional space.

**Review by Pawan-MSFT — COMMENTED** [review]
Please remove additional spaces.

File: samples/msgext-search-sso-config/nodejs/bots/teamsMessagingExtensionsSearchAuthConfigBot.js (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/msgext-search-sso-config/nodejs/bots/teamsMessagingExtensionsSearchAuthConfigBot.js:379]: Changed it

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving the PR.

File: samples/msgext-search-sso-config/nodejs/appPackage/manifest.json [hunk 3]
```diff
@@ -94,8 +94,9 @@
           "type": "link",
           "value": {
             "domains": [
-              "*.botframework.com"
-            ]
+              "*.botframework.com",
+            ],
+			"supportsAnonymizedPayloads": true
```
Comment (Wajeed-msft) [inline samples/msgext-search-sso-config/nodejs/appPackage/manifest.json:99]: Please check and correct the indentation.

**Review by Wajeed-msft — APPROVED** [review]

File: samples/msgext-search-sso-config/nodejs/appPackage/manifest.json (hunk 3 above)
Comment (Prithvi-MSFT) [reply samples/msgext-search-sso-config/nodejs/appPackage/manifest.json]: Fixed the indentation

**Comment by ChetanSharma-msft** [comment]
**Executed build pipeline:** https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/3604664890
**Result:**

**Comment by ChetanSharma-msft** [comment]
**Executed build pipeline:** https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/3604664890
**Result:** 
image

**Pull request merged**